docs/environment.md

```diff
@@ -37,11 +40,114 @@ You can skip this step if you have installed WSL as per the previous item.
 Download a Conda or Miniconda [installer for your platform](https://docs.conda.io/en/latest/miniconda.html)
 and run it.
 
+## Creating a Conda environment
+Note that in order to create the Conda environment you will need to have build tools installed on your machine. If you are running Windows, they should be already installed with Conda distribution.   
+
+You can install build tools on Ubuntu (and Debian-based distributions) by running  
+`sudo apt-get install build-essential`  
+If you are running CentOS/RHEL distributions, you can install the build tools by running  
+`yum install gcc gcc-c++ kernel-devel make`
 
-## Create a Conda environment
 Start the `conda` prompt for your platform. In that prompt, navigate to your repository root and run
 `conda env create --file environment.yml`
 
+## Using GPU locally
+
+It is possible to run the training process on a local machine. It will not be as performant as using a GPU cluster that Azure ML offers and you will not be able to take advantage of other Azure ML features such as comparing run results, creating snapshots for repeatable machine learning experiments or keeping history of experiment runs. At the same time it could be useful to experiment with code or troubleshoot things locally. 
+
+The SDK uses PyTorch to compose and run DNN computations. PyTorch can run on any CPU, but it is also capable of taking advantage of the underlying GPU via NVidia CUDA technology, which accelerates computations dramatically. 
```

Comment:
I'd skip that, I assume everyone in the DNN space knows that.

Reply:
Rephrased